OpenSim/Analyses/JointReaction.h

```diff
@@ -72,12 +72,12 @@ OpenSim_DECLARE_CONCRETE_OBJECT(JointReaction, Analysis);
         /* Joint reference*/
         const Joint* joint;
         /* The body upon which the force is applied */
-        const PhysicalFrame* appliedOnBody;
+        const Frame* appliedOnBody;
         /* Is the reaction force applied on the Body the child or the parent
            of the joint?*/
         bool isAppliedOnChild;
         /* The reference Frame in which the force should be expressed */
-        const PhysicalFrame* expressedInFrame;
+        const Frame* expressedInFrame;
```

Comment:
Based on your updated implementation, I believe that his is right - that you can express the force in any Frame, physical or not. Should the property be updated and/or the lookup of the `Frame` from its name in the list property?